resources.go

```diff
@@ -1875,17 +1875,45 @@ func Provider() tfbridge.ProviderInfo {
 					"utils.ts",  // Helpers,
 				},
 				Modules: map[string]*tfbridge.OverlayInfo{
+					"acmpca": {
+						DestFiles: []string{
+							"metrics.ts",
```

Comment:
general pattern will be that packages expose appropriate metrics through the 'metrics.ts' file/module.